torchao/quantization/quant_api.py

```diff
@@ -829,6 +847,31 @@ def apply_float8_static_activation_quant(weight: torch.Tensor):
     return _get_linear_subclass_inserter(apply_float8_static_activation_quant)
 
 
+# TO DISCUSS:
+# below currently just band aids float8 training in without actually changing 
+# how float8 training is configured. Is this fine? Is this a no-go? Somewhere in the
+# middle? How strongly do we care about config consistency among various 
+# projects? 
+# Vasiliy: I'm cool to conform to some other config standard, I would just say 
+# that a config (and not callables/partials) does seem more natural for 
+# training as the # of things to configure is significantly higher compared to
+# your average inference setup.
+def float8_for_training(
```

Comment:
I think it would be cool if float8_for_training had usage options for both the config and non-config use cases. I don't know if thats feasible but i think having config is fine, it'd be a little better for random OSS users if there was a more english readable way to set it up.

or maybe its just down to some helpers? I think at the least we should have something like 

quantize_(mod, float8_for_training(config=get_default_config(dtype=fp16)))